cmds/ocm/commands/ocmcmds/common/addhdlrs/comp/elements.go

```diff
@@ -155,6 +155,10 @@ type ResourceSpec struct {
 
 var _ addhdlrs.ElementSpec = (*ResourceSpec)(nil)
 
+func (r *ResourceSpec) GetRawIdentity() metav1.Identity {
+	return metav1.NewIdentity(r.Name, metav1.SystemIdentityVersion, r.Version)
```

Comment:
Why is this constant called `SystemIdentityVersion`?

Reply:
SystemIdentityVersion and SystemIdentityName are the two predefined identity properties use to identify elements (resources, sources, referebces) in CDs.

Reply:
added docu to their const definitions.